.gitignore

```diff
@@ -20,3 +20,7 @@ clangs
 bin/test
 bin/unittest
 .DS_Store
+*.so.*
+*.so
+/bin/*
+*~
```

Comment:
Ohk.  Will check.
Actually i just added everything i found git added when i did 'git add .'
Will change it.